server/src/routes/api/v1/activityRouter.js

```diff
@@ -9,11 +9,13 @@ activityRouter.use("/:id/reviews", activityReviewRouter)
 
 activityRouter.get("/:id", async (req, res) => {
     const activityId = req.params.id
-
+    const { user } = req.session.passport
     try {
+        console.log(user)
         const activity = await Activity.query().findById(activityId)
         const serializedActivity = await ActivitySerializer.getSummaryOfOne(activity)
-        return res.status(200).json({ activity: serializedActivity })
+        const fullActivity = {...serializedActivity, user}
```

Comment:
not sure why the user needs to also be sent? the endpoint is for Activities, so ideally we should only be sending back the activity data. The React app is always going to fetch the currentUser from the App component, and can pass that information as needed to other components, eliminating the need for us to additionally query/ send that data. 

when we _do_ need the current user in the back end here, it should be accessed via `req.user` (not the passport session above)
https://learn.launchacademy.com/teams/admins/curricula/on-campus-boston-41/lesson_groups/weeks_7_&_8:_group_project/lessons/authentication-with-react-and-passport